.gitignore

```diff
@@ -48,4 +48,3 @@ read-badger
 read-protocol-state
 remove-execution-fork
 util
-bootstrap
```

Comment:
there is anew `make` option which generates the `bootstrap` util hence we need to retain this